sysapi/sysapi/GetRpBuffer.c

```diff
@@ -44,8 +45,32 @@ TSS2_RC Tss2_Sys_GetRpBuffer(
         ctx->rval != TSS2_RC_SUCCESS)
         return TSS2_SYS_RC_BAD_SEQUENCE;
 
-    *rpBuffer = ctx->rpBuffer;
-    *rpBufferUsedSize = ctx->rpBufferUsedSize;
+    /* Calculate the position of the response parameter section within the TPM
+     * repsponse as well as its size. Structure is:
+     * Header (tag, responseSize, responseCode)
+     * handle(if Command has handles)
+     * parameterSize (if TPM_ST_SESSIONS), size of rpArea
+     * rpArea
+     * Sessions (if TPM_ST_SESSIONS) */
+	size_t offset = 0;
```

Comment:
No need to assign it to 0, just offset = sizeof(TPM20_Header_Out); below

Reply:
ok, sure